teps/0096-pipelines-v1-api.md

```diff
@@ -235,6 +235,8 @@ This policy should be updated to include Tekton metrics as part of the API. No o
 #### API Changes
 
 - Fix [pain points](https://github.com/tektoncd/pipeline/issues/3792) associated with TaskRun and PipelineRun Status.
+- Rename `PipelineRun.TaskRunSpecs[].TaskServiceAccountName` to `PipelineRun.TaskRunSpecs[].ServiceAccountName`.
+- Rename `PipelineRun.TaskRunSpecs[].TaskPodTemplate` to `PipelineRun.TaskRunSpecs[].PodTemplate`.
```

Comment:
It's a nit but, this is "code", not api change. 
The api change here is renaming `spec.taskRunSpecs.taskServiceAccountName` to …, and `spec.taskRunSpecs.taskPodTemplate` to ….